grammar.js

```diff
@@ -698,37 +701,37 @@ module.exports = grammar({
       $.table_reference,
       choice(
         seq(
-          choice(
-            $.add_column,
-            $.alter_column,
-            $.drop_column,
-          ),
+          $._alter_specifications,
           repeat(
             seq(
               ",",
-              choice(
-                $.add_column,
-                $.alter_column,
-                $.drop_column,
-              )
+              $._alter_specifications
             )
           )
         ),
-        // some operations may not be chained to others
```

Comment:
Is this still valid or was that just an artifact of another dialect of SQL?

Reply:
[fix: Lift ALTER TABLE combination restriction (only Postgres)](https://github.com/DerekStride/tree-sitter-sql/commit/be20ed053a6d0a6125e485f4796a488f91df987d)
Resolves comment: https://github.com/DerekStride/tree-sitter-sql/issues/82#issuecomment-1416227381